src/__internal__/input/input-presentation.component.tsx

```diff
@@ -29,6 +29,8 @@ export interface CommonInputPresentationProps extends ValidationProps {
   size?: Sizes;
   /** If true, the component has an icon rendered inside */
   hasIcon?: boolean;
+  /** Specify a custom border radius. Any valid CSS string */
+  borderRadius?: string;
```

Comment:
Comment: I wonder if we're better off just copying how this was done in the `Box` component for now rather than offering tis level of granular control, maybe a question for @harpalsingh though

https://github.com/Sage/carbon/blob/129cebf1b0d82ea29334d5d769fdac1421c9803c/src/components/box/box.component.tsx#L27
https://github.com/Sage/carbon/blob/129cebf1b0d82ea29334d5d769fdac1421c9803c/src/components/box/box.component.tsx#L61

Reply:
@harpalsingh has requested we limit to just tokens. I'll get this implemented now 👍🏻